packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -834,16 +874,30 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval>
 	 * @param id - Id of the interval whose properties should be changed
 	 * @param props - Property set to apply to the interval. Shallow merging is used between any existing properties
 	 * and `prop`, i.e. the interval will end up with a property object equivalent to `{ ...oldProps, ...props }`.
+	 * @deprecated - call IntervalCollection.change and specify properties
 	 */
 	changeProperties(id: string, props: PropertySet);
 	/**
 	 * Changes the endpoints of an existing interval.
 	 * @param id - Id of the interval to change
-	 * @param start - New start value. This can be the existing position to keep it unchanged.
-	 * @param end - New end value. This can be the existing position to keep it unchanged.
+	 * @param start - New start value, if defined. `undefined` signifies this endpoint should be left unchanged.
+	 * @param end - New end value, if defined. `undefined` signifies this endpoint should be left unchanged.
+	 * @param props - Property set to apply to the interval. Shallow merging is used between any existing properties
+	 * and `prop`, i.e. the interval will end up with a property object equivalent to `{ ...oldProps, ...props }`.
```

Comment:
This looks like a cut-and-paste or merge error: props is not a param here, and start and end can't be undefined.